core/contracts/oracle/implementation/Store.sol

```diff
@@ -132,6 +132,7 @@ contract Store is StoreInterface, MultiRole, Withdrawable {
      * @param newWeeklyDelayFee fee escalation per week of late fee payment.
      */
     function setWeeklyDelayFee(FixedPoint.Unsigned memory newWeeklyDelayFee) public onlyRoleHolder(uint(Roles.Owner)) {
+        require(newWeeklyDelayFee.isLessThan(1), "weekly delay fee must be <= 100%");
```

Comment:
nit: require message says `<=`, but you test `<`.

Not sure whether the other checks in the store are < or <=, but we should probably make them match just for consistency.